Tasks/google-play-release/GooglePlay.js

```diff
@@ -27,6 +27,17 @@ if (authType === "JsonFile") {
 }
 
 var apkFile = resolveGlobPath(tl.getPathInput("apkFile", true));
+var apkFileList = [apkFile];
+var additionaApks = tl.getDelimitedInput('additionalApks', '\n');
+if (additionaApks.length > 0) {
+    for (var i in additionaApks) {
+        apkFileList.push(resolveGlobPath(additionaApks[i]));
```

Comment:
Currently, `resolveGlobPath` will only grab the first item that matches the specified pattern. Does that make sense in the context of the additional APKs input?